tests/integration/reqs/test_generic_request.py

```diff
@@ -4,13 +4,11 @@
 
 
 class TestGenericRequest(IntegrationTestCase):
+    # Note: Support for the tx_history command has been removed since rippled API v2
     @test_async_and_sync(globals())
     async def test_constructor(self, client):
         response = await client.request(
-            GenericRequest(
-                method="tx_history",
-                start=0,
-            )
+            GenericRequest(method="tx_history", start=0, api_version=1)
```

Comment:
There should be an additional test for another request that makes sure that the API version is handled properly - perhaps `account_info` since there is a significant difference in the shape of the output.